lib/action_cable_client.rb

```diff
@@ -22,7 +22,8 @@ class Commands
   attr_reader :_message_factory
   # The queue should store entries in the format:
   # [ action, data ]
-  attr_accessor :message_queue, :_subscribed, :_subscribed_callback, :_pinged_callback, :_connected_callback
+  attr_accessor :message_queue, :_subscribed
```

Comment:
what's the convention now a days? multiple attr_accessor's or just a multi-line one? like: 

```ruby
attr_accessor :message_queue, :_subscribed, 
              :_subscribed_callback, :_pinged_callback, 
              :_connected_callback, :_disconnected_callback
```

Reply:
IFAIK, there is no strict rule about this.
I split these lines because that was a too long line for `rubocop`. To improve readability, I move callbacks one another line.